spanner/spanner_samples.rb

```diff
@@ -113,12 +131,21 @@ def create_database_with_version_retention_period project_id:, instance_id:, dat
   # database_id = "Your Spanner database ID"
 
   require "google/cloud/spanner"
+  require "google/cloud/spanner/admin/database"
 
-  spanner  = Google::Cloud::Spanner.new project: project_id
-  instance = spanner.instance instance_id
-  version_retention_period = "7d"
+  database_admin_client = Google::Cloud::Spanner::Admin::Database.database_admin project_id: project_id
+
+  instance_path = database_admin_client.instance_path project: project_id, instance: instance_id
```

Comment:
So we don't store project_id in `database_admin_client`, right? We have to pass `project_id` again here.

Reply:
yes, we can remove it from client creation :+1: